index.html

```diff
@@ -1 +1,147 @@
+                <li class="list"><a href="#program">Program</a></li>
+                <li class="list"><a href="#about">Join</a></li>
+                <li class="list"><a href="#about">Sponsor</a></li>
+                <li class="list"><a href="#about">News</a></li>
+            </ul>
+        </div>
+        <nav>
+            <div id="hamburger">
+                <i class="fa-sharp fa-solid fa-bars"></i>
+            </div>
+            <div id="nav-container">
+                <div id="top-bar">
+                    <ul class="top-bar-lists">
+                        <li class="top-bar-list"><a href="https://facebook.com"><img class="handle1" src="https://summit.cckorea.org/images/base/facebook.png" alt="facebook page"></a></li>
+                        <li class="top-bar-list"><a href="https://twitter.com"><img class="handle2" src="	https://summit.cckorea.org/images/base/twitter.png" alt="twitter page"></a></li>
+                        <li class="top-bar-list"><a href="#English">English</a></li>
+                        <li class="top-bar-list"><a href="#Mypage">My Page</a></li>
+                        <li class="top-bar-list"><a href="#logout">Logout</a></li>
+                    </ul>
+                </div>
+                <div id="nav-bar">
+                    <div class="nav-bar-container">
+                        <div id="top-logo">
+                            <img src="/assets/logo.png" alt="logo" id="desktop-logo">
+                        </div>
+                        <ul id="nav-lists">
+                            <li class="nav-list"><a href="/about.html">About</a></li>
+                            <li class="nav-list"><a href="#program">Program</a></li>
+                            <li class="nav-list"><a href="#about">Join</a></li>
+                            <li class="nav-list"><a href="#about">Sponsor</a></li>
+                            <li class="nav-list"><a href="#about">News</a></li>
+                            <li class="nav-list campaign"><a class="campaign-txt" href="#campaign">CC Campaign</a></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <section id="headline">
+        <div class="headline-container">
+            <div class="text-container">
+                <h2 class="welcome">"Welcome"</h2>
+                <div class="main-header">
+                    <h1 class="h1_1">Learn It...</h1>
+                    <h1 class="h1_2">Solve It...</h1>
+                </div>
+                <p class="description">An educative learning app for both students and teachers. This app Contains slides and questions on general knowledge as well as engineering</p>
+                <p class="quote">"Failure is an option here. If things are not failing, you are not innovating enough."</p>
+                <p class="elon">(Elon Musk)</p>
+            </div>
+        </div>
```

Comment:
[REQUIRED] Good job reducing the length of the landing page but now it is too short and still not close to the template design as required. I will suggest that you add a similar or the same amount of text as the one in the template and style the size, width and height of the text to match the template.

![image](https://user-images.githubusercontent.com/62938735/226114005-b41fc368-4887-4629-8b4e-9899f87a8196.png)